test/test-analytics.js

```diff
@@ -119,7 +68,7 @@ describe('analytics', function () {
 
 						res.writeHead(204);
 						res.end();
-						setTimeout(cleanup, 100);
+						setTimeout(cleanup, 10);
```

Comment:
Timeout of 100 was too long on my machine and the child would exit before this had ran